source/assets/public/index.css

```diff
@@ -72,25 +74,29 @@ body {
 .breakpoint-desktop-up .container {
   padding-left: 15rem;
 }
+
+/* Navigation */
+
+.nav {
+  line-height: 3rem;
+  padding-bottom: 1.5rem;
+  padding-right: 1rem;
+  text-align: right;
```

Comment:
I think that could be a nice idea and would help keep it noticeable  when stickie'd you may also want to centre it or reduce it to 'hamburger' form which pops out on mobile.